### PR TITLE
Feature/vehicle details page 

### PR DESCRIPTION
**Pull request by rmuehler**
image

### CODE REVIEW

File: src/app/modules/profile/services/vehicle-submission.service.ts [hunk 1]
```diff
@@ -80,3 +54,37 @@ export class VehicleSubmissionService {
...
+//
+// updateStatus(vin: string, status: number): void {
+//   this.getByVIN(vin).subscribe((data) => {
+//     data.status = status;
+//   });
+// }
+//
+// getAll(): Observable<Submission[]> {
+//   return of(this.vehicleSubmissions);
+// }
```
Comment (GannonDeWalt) [inline src/app/modules/profile/services/vehicle-submission.service.ts:90]: Do we need this commented out code?

Comment (rmuehler) [reply src/app/modules/profile/services/vehicle-submission.service.ts]: removed

**Review by GannonDeWalt — APPROVED** [review]
Looks good

**Review by ammitchky — APPROVED** [review]
looks good

**Pull request merged**